arm/Microsoft.Network/virtualWans/deploy.bicep

```diff
@@ -9,41 +9,16 @@ param name string
   'Standard'
   'Basic'
 ])
-param virtualWanSku string = 'Standard'
+param type string = 'Standard'
```

Comment:
I'd suggest changing the type description to "Optional. The type of the VirtualWAN."

Reply:
done